src/relayFeeCalculator/chain-queries/polygon.ts

```diff
@@ -43,3 +43,17 @@ export class PolygonQueries extends QueryBase {
     return Number((tokenPrice / maticPrice).toFixed(this.symbolMapping["MATIC"].decimals));
   }
 }
+
+export class PolygonMumbaiQueries extends PolygonQueries {
+  constructor(
+    provider: providers.Provider,
+    symbolMapping = TOKEN_SYMBOLS_MAP,
+    spokePoolAddress = getDeployedAddress("SpokePool", CHAIN_IDs.MUMBAI),
+    simulatedRelayerAddress = "0x9A8f92a830A5cB89a3816e3D267CB7791c16b04D",
```

Comment:
Is there a constant EOA we can use?

Reply:
Defined here: https://github.com/across-protocol/sdk-v2/pull/447/commits/c4e8d8e385666cc9289903dcb344b20d127a8969